packages/codegen/templates/app00/package.json

```diff
@@ -1,24 +1,36 @@
 {
-  "name": "@app/{{ name }}",
+  "name": "@app/{{name}}",
   "version": "1.0.0",
   "private": true,
   "scripts": {
     "test": "jest",
-    "dev": "node --trace-warnings ./../../bin/run-keystone-app.js",
+    "prepare": "node ./bin/prepare.js",
+    "dev": "yarn build:deps && yarn dev:start",
+    "dev:start": "node --trace-warnings ./../../bin/run-keystone-app.js",
     "debug": "node inspect node_modules/.bin/keystone dev",
+    "build:deps": "turbo build --filter={{name}}^... --filter=!condo",
     "build": "keystone build",
     "start": "NODE_ENV=production node ./../../bin/run-keystone-app.js",
+    "worker": "cross-env PHASE=worker node worker.js",
     "kmigrator": "./../../bin/kmigrator.py",
     "makemigrations": "./../../bin/kmigrator.py makemigrations",
     "migrate": "./../../bin/kmigrator.py migrate",
     "migrate:down": "./../../bin/kmigrator.py down",
-    "maketypes": "mkdir -p dist/admin && NODE_ENV=test node ./../../bin/create-graphql-schema.js && graphql-codegen -c codegen.yaml",
+    "migrate:unlock": "./../../bin/kmigrator.py unlock",
+    "maketypes:{{name}}": "mkdir -p dist/admin && NODE_ENV=test node ./../../bin/create-graphql-schema.js StitchSchemaMiddleware && graphql-codegen -c codegen.yaml",
```

Comment:
For what add `StitchSchemaMiddleware` here when run `node ./../../bin/create-graphql-schema.js`?

Reply:
For exclude StitchSchemaMiddleware from keystone app in `create-graphql-schema.js` script.
There are a error throws if run app maketypes with StitchSchemaMiddleware middleware.
The same `maketypes` is in condorb or registry apps

Reply:
Should we exclude this in other miniapps?

Reply:
I think no, this error throws only in first time, when `.schema.graphql` file not found or it's empty.
Error in makeGatewaySchema -> loadSchema